captcha/templates/captcha/widget_nocaptcha.html

```diff
@@ -0,0 +1,33 @@
+<script src='https://www.google.com/recaptcha/api.js'></script>
```

Comment:
Language setting should be set as `hl` GET param according the new [docs ](http://developers.google.com/recaptcha/docs/display#config)